### DISCUSSION TITLE
issue regarding imports

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1835] [General]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/4fae219e-1d53-40e0-b8e4-d3ece5225b54)
I am getting this error in my defi protocol lesson -- Source "lib/openzeppelin-contracts/contracts/security/ReentrancyGuard.sol" not found: File not found. Searched the following locations: "".
I installed openzeppelin package
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/791aec8d-790a-4352-ab70-437f114a66af)

**Comment by EngrPips** [comment]
Hello @panditdhamdhere, They might have changed the location of the file in their new version, if you are on Mac, use `command + p` to search for the file, and then you should use the path that you found to import the file if you are on windows you can `cltr + p` instead

**Reply by panditdhamdhere** [reply]
Thank you very much, it is really helpful! problem solved

**Reply by EngrPips** [reply]
Fantastic. Keep crushing it, friend.
